lib/remote-api-caller-helper.ts

```diff
@@ -0,0 +1,59 @@
+import { CryptoUtil } from "./crypto-util.js";
+import { RemoteAPICaller } from "./gmd-api-caller.js";
+
+
+
+
+export class RemoteAPICallerHelper extends RemoteAPICaller {
+    //Latest block
+    async getBlockNumber(): Promise<number> {
+        const data = await this.apiCall('get', { requestType: 'getBlock' } as Record<string, string>)

```

Comment:
So here you can do `this.apiCall<{height: number}>('get', ...)` and it will know to cast to number if the function had a generic <T = unknown>

Reply:
Will do, thanks.